frappe/email/doctype/email_account/email_account.json

```diff
@@ -419,7 +419,7 @@
  ],
  "icon": "fa fa-inbox",
  "links": [],
- "modified": "2020-05-11 15:18:43.931499",
+ "modified": "2020-07-18 23:11:13.497087",
```

Comment:
could you reset this file to its previous state? ideally there should be no changes to this file.

Reply:
same for email_account.py

Reply:
Ok cool 👍